release/release_tests.yaml

```
@@ -2113,34 +2113,54 @@
     cluster_compute: compute_configs/compute_gpu_4x4_aws.yaml
 
   variations:
-    - __suffix__: full_training
+    - __suffix__: full_training_parquet
       run:
         timeout: 2000
-        script: RAY_TRAIN_V2_ENABLED=1 python train_benchmark.py --task=image_classification --dataloader_type=ray_data --num_workers=16
+        script: RAY_TRAIN_V2_ENABLED=1 python train_benchmark.py --task=image_classification_parquet --dataloader_type=ray_data --num_workers=16
 
-    - __suffix__: full_training_torch_dataloader
+    - __suffix__: full_training_parquet_torch_dataloader
```

Nit: Maybe naming convention can be "." between each sweepable config?

```suggestion
    - __suffix__: full_training.parquet.torch_dataloader
```